src/index.js

```diff
@@ -0,0 +1,130 @@
+import 'regenerator-runtime/runtime';
+import axios from 'axios';
```

Comment:
These were necessary for deployment, but when these imports run locally it breaks the site. When you have conditional code needed for one environment, but not another, you can create a separate branch for deployment that has these specific values and deploy the separate branch. That way you have a main branch that works locally and a deployment branch that has everything it needs to get deployed